packages/react-output-target/src/output-react.ts

```diff
@@ -68,17 +96,30 @@ import { createReactComponent } from './react-component-lib';\n`;
     typeImports,
     sourceImports,
     registerCustomElements,
-    components.map(createComponentDefinition).join('\n'),
+    components.map(cmpMeta => createComponentDefinition(cmpMeta, outputTarget.includeImportCustomElements)).join('\n'),
   ];
 
   return final.join('\n') + '\n';
 }
 
-function createComponentDefinition(cmpMeta: ComponentCompilerMeta) {
+/**
+ * Defines the React component that developers will import
+ * to use in their applications.
+ */
+export function createComponentDefinition(cmpMeta: ComponentCompilerMeta, includeCustomElement: boolean = false) {
   const tagNameAsPascal = dashToPascalCase(cmpMeta.tagName);
+  const importAs = (includeCustomElement) ? tagNameAsPascal + 'Cmp' : undefined;
+
+  let template = `export const ${tagNameAsPascal} = /*@__PURE__*/createReactComponent<${IMPORT_TYPES}.${tagNameAsPascal}, HTML${tagNameAsPascal}Element>('${cmpMeta.tagName}'`;
+
+  if (importAs) {
+    template += `, ${importAs}`;
+  }
```

Comment:
Not required, just an alternative
```suggestion
  let template = `export const ${tagNameAsPascal} = /*@__PURE__*/createReactComponent<${IMPORT_TYPES}.${tagNameAsPascal}, HTML${tagNameAsPascal}Element>('${cmpMeta.tagName}'`;

  if (includeCustomElement) {
    template += `, ${tagNameAsPascal}Cmp`;
  }
```